### ISSUE TITLE
Keras Implementation

### ISSUE THREAD

**Issue by 0xnurl** [ozanarkancan/char-ner#2]
Hi, 
I added a Keras implementation of your great model:
https://github.com/0xnurl/keras_character_based_ner

Only the Viterbi decoder is missing for now.

**Comment by dongfang91** [comment]
Hi,

I was looking at your code written in theano and Lasagne, it seems the model is not doing backpropagation through the viterbi decoder right?

**Comment by ozanarkancan** [comment]
Hi,

Yes, you are right. We use viterbi only in inference time. We train the network using regular cross-entropy loss.
